internal/git/git.go

```diff
@@ -22,36 +24,48 @@ func CloneCommandFor(s rukpakv1alpha1.GitSource) (string, error) {
 	if err != nil {
 		return "", err
 	}
-	return cmd.String(), nil
+	return cmd.String()
 }
 
-func (c *checkoutCmd) String() string {
+func (c *checkoutCmd) String() (string, error) {
 	var checkoutCommand string
 	var repository = c.Repository
 	var directory = c.Directory
 	var branch = c.Ref.Branch
 	var commit = c.Ref.Commit
 	var tag = c.Ref.Tag
+	var preset = ""
```

Comment:
```suggestion
	var preset string
```

Reply:
Thanks!  I'll update.

Reply:
That makes sense -- Go assigns a default zero value to all variables when they're declared (and not initialized to something) so something like
```
var preset string
```
results in `preset` equaling the default string `""` value automatically. Declaring it this way here is good because it's more clear IMO. Checkout https://go.dev/tour/basics/12 for a demo. Using the default zero types is generally very helpful and makes code clearer.

Reply:
Yeah +1 to using the default value here as well. We can initialize the `preset` variable to be an empty string by default, and if the c.SSLNoVerify option is true, override the default empty string value with the sslNoVerify value.